chroot_setup

```diff
@@ -0,0 +1,434 @@
+  user = chown_command[0]
+  chown_directory = f"{args.chroot_directory}{chown_command[1]}"
+  for root, dirs, files in os.walk(f"{args.chroot_directory}{chown_directory}"):
+    for dir in dirs:
+      printf(f"os.chown({os.path.join(root, dir)}, {user})")
+      if not CHROOT_DEBUG:
+        os.chown(os.path.join(root, dir), user)
+    for file in files:
+      printf(f"os.chown({os.path.join(root, file)}, {user})")
+      if not CHROOT_DEBUG:
+        os.chown(os.path.join(root, file), user)
+
+# :system
+# Executes a line from a .lst file using the /QOpenSys/usr/bin/system utility
+def chroot_system(line: str):
+  line = line.rstrip()
+  print(f"system -i {line}")
+  if not CHROOT_DEBUG:
+    os.system(f"system -i {line}")
+
+# :sh
+# Executes a line from a .lst file using Pythons os.system function, which is
+# run "by calling the Standard C function system()."
+def chroot_sh(line: str):
+  print(line)
+  if not CHROOT_DEBUG:
+    os.system(line)
+
+# :file
+# Parses and executes additional .lst files
+def chroot_setup (chroot_lst):
+  lst_file = open(chroot_lst, 'r')
+  action = None
+
+  # Iterate over all lines in the file, executing actions where encountered
+  for line in lst_file:
+    line = line.strip()
+
+    # Skip blank lines and comments
+    if not line or line.startswith('#'):
+      continue
+
+    # Determine action
+    elif line.startswith(':'):
+      action = actions.get(line, None)
+
+    # Execute action
+    else:
+      if action is not None:
+        action(line)
+
+# Mapping the .lst action keys to the functions that run on their values
+actions = {
+  ':file'       : chroot_setup,
+  ':mkdir'      : chroot_mkdir,      # done
+  ':ln_fix_rel' : chroot_ln_fix_rel, # done
+  ':ln_rel'     : chroot_ln_rel,     # done
+  ':ln'         : chroot_ln,         # done
+  ':mknod'      : chroot_mknod,      # done
+  ':cp_dir'     : chroot_cp_dir,     # done
+  ':cp'         : chroot_cp,         # done
+  ':chmod_dir'  : chroot_chmod_dir,  # done
+  ':chmod'      : chroot_chmod,      # done
+  ':chown_dir'  : chroot_chown_dir,  # done
+  ':chown'      : chroot_chown,      # done
+  ':system'     : chroot_system,     # done
+  ':sh'         : chroot_sh          # done
+}
+
+# Run an initial check on the environment, ensuring that we are running on IBM i
+# and setting environment variables
+def initial_check():
+  if (os.uname().sysname == 'OS400'):
+    # CHROOT_DEBUG = False
+    PATH='/QOpenSys/pkgs/bin:/QOpenSys/pkgs/sbin:/QOpenSys/usr/bin:/QOpenSys/usr/sbin:'
+    os.environ['PATH'] = PATH
+    print("**********************")
+    print("Live IBM i session (changes made).")
+    print("**********************")
+
+    if (args.verbose):
+      print(f"PATH={PATH}")
+
+  else:
+    # CHROOT_DEBUG = True
+    print("**********************")
+    print("Not IBM i, no action is taken (debug flow purpose only).")
+    print("**********************")
+    # exit -1
```

Comment:
This is the else leg that indicates we are not running chroot setup on IBM i therefor we should exit after printing the error message.

Reply:
Fixed this up in dd947c5